classes/PodsI18n.php

```diff
@@ -196,6 +198,10 @@ private static function default_strings() {
 
 			'%s items'                                     => __( '%s items', 'pods' ),
 
+			'You can only select %s item'                  => __( 'You can only select %s item', 'pods' ),
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 1243a0622531814d7d96ce6f85541929</sub>